docs/book/quick-start.md

```diff
@@ -956,8 +956,6 @@ to define all of these?
 Annotations allow us to solve this problem. You can define the following
 behaviors with the shipped annotations in laminas-form:
 
-- `AllowEmpty`: mark an input as allowing an empty value. This annotation does
-  not require a value.
```

Comment:
This feature is still present in laminas-inputfilter and there are also no plans to change this in the _near_ future.

Reply:
@froschdesign Good point, I'll restore `AllowEmpty` and `ContinueIfEmpty`.